service/src/main/java/org/mskcc/cmo/metadb/service/util/RequestDataFactory.java

```diff
@@ -86,8 +87,12 @@ private static RequestMetadata extractRequestMetadataFromJson(String requestMeta
         if (requestMetadataMap.containsKey("samples")) {
             requestMetadataMap.remove("samples");
         }
-        RequestMetadata requestMetadata = new RequestMetadata(
-                requestMetadataMap.get("requestId").toString(),
+        // an attempt to make this method flexible with both schemas
+        String requestId = ObjectUtils.firstNonNull(
+                String.valueOf(requestMetadataMap.get("igoRequestId")),
```

Comment:
[requested change] reverse the ordering of the fields checked

Reply:
👍